src/clustermove.cpp

```diff
@@ -43,7 +43,7 @@ void Cluster::_to_json(json &j) const {
     }
 }
 void Cluster::_from_json(const json &j) {
-    assertKeys(j, {"dp", "dprot", "dir", "threshold", "molecules", "repeat", "satellites", "dirrot"});
+    assertKeys(j, {"dp", "dprot", "dir", "threshold", "molecules", "repeat", "satellites", "spread", "dirrot"});
```

Comment:
You can delete this line completely as it's already covered by `docs/schema.yml`